ui/components/app/app-components.scss

```diff
@@ -5,8 +5,8 @@
 @import 'alerts/alerts';
 @import 'beta-header/index';
 @import 'cancel-speedup-popover/index';
-@import 'confirmations/alerts/alert-modal/index';
-@import 'confirmations/alerts/inline-alert/index';
+@import 'alerts-system/alert-modal/index';
```

Comment:
Would `alert-system` be a cleaner name for the parent directory instead of `alerts-system` given the two `s` next to each other?

Reply:
Done.